queries/models/financeiro/subsidio_faixa_servico_dia_tipo_viagem.sql

```diff
@@ -0,0 +1,158 @@
+  )
+}}
+
+WITH
+  subsidio_faixa_dia AS (
+  SELECT
+    data,
+    tipo_dia,
+    faixa_horaria_inicio,
+    faixa_horaria_fim,
+    consorcio,
+    servico,
+    pof
+  FROM
+    -- rj-smtr-dev.financeiro.subsidio_faixa_servico_dia
+    {{ ref("subsidio_faixa_servico_dia") }}
+  WHERE
+    data BETWEEN DATE("{{ var("start_date") }}")
+    AND DATE("{{ var("end_date") }}")
+  ),
+  servico_km_apuracao AS (
+  SELECT
+    data,
+    servico,
+    CASE
+      WHEN tipo_viagem = "Nao licenciado" THEN "Não licenciado"
+      WHEN tipo_viagem = "Licenciado com ar e autuado (023.II)" THEN "Autuado por ar inoperante"
+      WHEN tipo_viagem = "Licenciado sem ar" THEN "Licenciado sem ar e não autuado"
+      WHEN tipo_viagem = "Licenciado com ar e não autuado (023.II)" THEN "Licenciado com ar e não autuado"
+      ELSE tipo_viagem
+    END AS tipo_viagem,
+    id_viagem,
+    distancia_planejada,
+    subsidio_km,
+    -- subsidio_km_teto,
+    4.04 AS subsidio_km_teto,
+    indicador_viagem_remunerada
+  FROM
+    {{ ref("viagens_remuneradas") }}
+    -- rj-smtr.dashboard_subsidio_sppo.viagens_remuneradas
+  WHERE
+    data BETWEEN DATE("{{ var("start_date") }}")
+    AND DATE("{{ var("end_date") }}")
+  ),
+  indicador_ar AS (
+  SELECT
+    data,
+    id_veiculo,
+    status,
+    SAFE_CAST(JSON_VALUE(indicadores,"$.indicador_ar_condicionado") AS BOOL) AS indicador_ar_condicionado
+  FROM
+    -- {{ ref("sppo_veiculo_dia") }}
+    rj-smtr.veiculo.sppo_veiculo_dia
+  WHERE
+    data BETWEEN DATE("{{ var("start_date") }}")
+    AND DATE("{{ var("end_date") }}")
+  ),
+  viagem AS (
+  SELECT
+    data,
+    servico_realizado AS servico,
+    id_veiculo,
+    id_viagem,
+    datetime_partida
+  FROM
+    -- {{ ref("viagem_completa") }}
+    rj-smtr.projeto_subsidio_sppo.viagem_completa
+  WHERE
+    data BETWEEN DATE("{{ var("start_date") }}")
+    AND DATE("{{ var("end_date") }}")
+  ),
+  ar_viagem AS (
+  SELECT
+    v.data,
+    v.servico,
+    v.id_viagem,
+    v.datetime_partida,
+    COALESCE(ia.indicador_ar_condicionado, FALSE) AS indicador_ar_condicionado
+  FROM
+    viagem v
+  LEFT JOIN
+    indicador_ar ia
+  ON
+    ia.data = v.data
+    AND ia.id_veiculo = v.id_veiculo
+  ),
+  subsidio_servico_ar AS (
+    SELECT
+      sfd.data,
+      sfd.tipo_dia,
+      sfd.faixa_horaria_inicio,
+      sfd.faixa_horaria_fim,
+      sfd.consorcio,
+      sfd.servico,
+      sfd.pof,
+      COALESCE(s.tipo_viagem, "Sem viagem apurada") AS tipo_viagem,
+      s.id_viagem,
+      s.distancia_planejada,
+      s.subsidio_km,
+      s.subsidio_km_teto,
+      s.indicador_viagem_remunerada,
+      CASE
+        WHEN s.tipo_viagem = "Autuado por ar inoperante" THEN TRUE
+        WHEN s.tipo_viagem = "Licenciado sem ar e não autuado" THEN TRUE
+        WHEN sfd.pof < 60 THEN TRUE
+        WHEN s.tipo_viagem = "Registrado com ar inoperante" THEN TRUE
+        ELSE FALSE
+      END AS indicador_penalidade_judicial,
```

Comment:
Substituir para `WHEN s.tipo_viagem IN ("Autuado por ar inoperante", "Licenciado sem ar e não autuado", "Registrado com ar inoperante") THEN TRUE`